src/adapters/elements-fetcher.ts

```diff
@@ -41,7 +41,7 @@ export type ElementsResult<T> = {
   totalAmount: number
 }
 
-export type ItemType = 'wearable' | 'emote' | 'smartWearable'
+export type ItemType = 'wearable' | 'emote' | 'smartWearable' | 'polygonWearables'
```

Comment:
I think it would be better to use another field called `network` and use `POLYGON` to fetch polygon wearables using item type and the network filters. WDYT?